packages/react-aria-components/docs/Table.mdx

```diff
@@ -149,15 +153,13 @@ import {Table, TableHeader, TableBody, Column, Row, Cell} from 'react-aria-compo
       outline-offset: -2px;
     }
 
-    &[data-pressed] .react-aria-Cell {
+    &[data-pressed] {
       background: var(--spectrum-global-color-gray-200);
     }
 
     &[data-selected] {
-      .react-aria-Cell {
-        background: var(--highlight-background);
-        color: var(--highlight-foreground);
-      }
+      background: var(--highlight-background);
+      color: var(--highlight-foreground);
 
```

Comment:
Looks like this was was actually broken in production for a while as well. The problem was that the outline was on the row, but we put the background on the cells which covered it. This moves the background back to the row instead. However, Firefox seems not to support border radius when rendering the background of a table row. Adding the `clip-path` as well works around this.